packages/tanstack-react-router-urql/src/useUrqlValue.ts

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { SSRContext } from './Provider';
+import { useRouter } from '@tanstack/react-router';
+import type { SerializedResult } from 'urql';
+
+function getStreamKey(operationKey: number) {
+  return `__URQL__${operationKey}`;
+}
+
+function useSsrExchange() {
+  const ssrExchange = React.useContext(SSRContext);
+
+  if (!ssrExchange) {
+    throw new Error(
+      'Missing "UrqlProvider" component as a parent or did not pass in an "ssrExchange" to the Provider.'
+    );
+  }
+  return ssrExchange;
+}
+
+export function useStreamUrqlValue(operationKey: number): void {
```

Comment:
Name seems a bit convoluted, useStreamedValue and useStreamValue is better as it doesn't even leave the boundaries of our package

Reply:
done